packages/extension-utils/src/authUtils.ts

```diff
@@ -108,9 +115,46 @@ export class OAuthConfigProvider extends ApiSettings {
       ...{
         base_url,
         looker_url,
-        client_id: 'looker.api-explorer',
+        client_id: this.clientId,
         redirect_uri: `${window.location.origin}/oauth`,
       },
     }
   }
 }
+
+export interface InitSdkSettings {
+  /** agent tag to use for the SDK requests */
+  agentTag: string
+  /** Local storage key for retrieving auth configuration settings */
+  configKey: string
+  /** Id used to register this application on the Looker server */
+  clientId: string
+  /** A callback function that returns an sdk instance */
+  createSdkCallback: (session: BrowserSession) => APIMethods
```

Comment:
Can this return `IAPIMethods` instead? Is there something in `APIMethods` that's not documented in `IAPIMethods`?